src/merge.ts

```diff
@@ -0,0 +1,135 @@
+import type { Config, Octokit, File, FrontMatter } from './types';
+import type { Repository } from '@octokit/webhooks-types';
+import localConfig from './localConfig';
+import fm from 'front-matter';
+import yaml from 'js-yaml';
+
+async function generateEIPNumber(octokit: Octokit, repository: Repository) {
+    // Get all EIPs
+    const eips = (await octokit.rest.repos.getContent({
+        owner: repository.owner.login,
+        repo: repository.name,
+        path: 'EIPS'
+    })).data as any[];
+
+    // Get all EIP numbers
+    const eipNumbers = eips
+        .filter(eip => eip.name.startsWith('eip-'))
+        .map(eip => Number(eip.name.split('-')[1]));
```

Comment:
How would this handle something like `eip-an_extension_to_ERC-721.md`?

Reply:
It woukdn't.